app/src/styles/_components.scss

```diff
@@ -38,6 +38,28 @@
             margin-right: 20px; 
         }
 
+<<<<<<< HEAD
```

Comment:
looks good but you have to get rid of the old code, so idk which one of these is yours but you accidentally left in the merge conflicts